pom.xml

```diff
@@ -382,41 +382,44 @@
         <groupId>org.revapi</groupId>
         <artifactId>revapi-maven-plugin</artifactId>
         <configuration>
+          <failBuildOnProblemsFound>false</failBuildOnProblemsFound>
+          <pipelineConfiguration>
+            <transformBlocks>
+              <block>
+                <item>manually-vetted</item>
+                <item>revapi.versions</item>
+              </block>
+            </transformBlocks>
+          </pipelineConfiguration>
           <analysisConfiguration>
+            <justification>Shaded artefacts are not part of the public API</justification>
+            <criticality>allowed</criticality>
+            <revapi.reporter.json>
+              <minSeverity>EQUIVALENT</minSeverity>
+              <minCriticality>allowed</minCriticality>
+              <output>target/revapi-result.json</output>
+              <indent>false</indent>
+              <append>false</append>
+              <keepEmptyFile>true</keepEmptyFile>
+            </revapi.reporter.json>
             <revapi.differences id="manually-vetted">
-              <justification>Shaded artefacts are not part of the public API</justification>
-              <criticality>allowed</criticality>
+              <attachments>
+                <vetted>ok</vetted>
+              </attachments>
               <differences combine.children="append">
-                <item>
-                  <ignore>true</ignore>
-                  <regex>true</regex>
-                  <classQualifiedName>shaded.*</classQualifiedName>
-                  <code>.*</code>
-                </item>
-                <item>
-                  <ignore>true</ignore>
-                  <class>edu.hm.hafner.analysis.parser.checkstyle.TopicRule</class>
-                  <regex>true</regex>
-                  <code>.*</code>
-                </item>
-                <item>
-                  <ignore>true</ignore>
-                  <package>javax.xml.transform</package>
-                  <code>java.class.externalClassExposedInAPI</code>
-                </item>
-                <item>
-                  <!-- No part of codingstyle library -->
-                  <ignore>true</ignore>
-                  <regex>true</regex>
-                  <classQualifiedName>edu.hm.hafner.util.LookaheadStream</classQualifiedName>
-                  <oldArchive>edu.hm.hafner:analysis-model:jar.*</oldArchive>
-                  <newArchive>edu.hm.hafner:codingstyle:jar.*</newArchive>
-                  <code>.*</code>
-                </item>
```

Comment:
Are those ignores not required anymore?

Reply:
That was because I searched for the rule which creates a empty report. 
The problem was this item when this is in the pom with regex true nothing gets reported. 


           <item>
                  <ignore>true</ignore> 
                  <class>edu.hm.hafner.analysis.parser.checkstyle.TopicRule</class>
                  <regex>true</regex>
                  <code>.*</code>
           </item> 
                 
For ignoring classes or packages I think the filter option is better.
https://revapi.org/revapi-java/0.27.0/element-filters.html
As far as I understand is filtering better suited for ignoring elements / files / packages. I will push a suggestion.